components/Glance.vue

```diff
@@ -4,43 +4,19 @@ import GradientBox from '@/components/GradientBox';
 export default {
   components: { GradientBox },
   props:      {
-    provider: {
-      type:     String,
-      required: true
-    },
-    kubernetesVersion: {
-      type:     String,
-      required: true
-    },
-    totalNodes: {
-      type:     Number,
-      required: true
-    },
-    created: {
-      type:     String,
-      required: true
+    // array of slot names being used
+    slots: {
+      type:    Array,
+      default: () => []
```

Comment:
Not going to hold up the rest of the PR at this point, but I don't really love having to declare the slots you want and then say them again to use them (and hope they match) as an API.

Tabbed/Tab does this automatically by [providing](https://github.com/rancher/dashboard/blob/master/components/Tabbed/index.vue#L49-L67), [injecting](https://github.com/rancher/dashboard/blob/master/components/Tabbed/Tab.vue#L3), and [calling](https://github.com/rancher/dashboard/blob/master/components/Tabbed/Tab.vue#L37-L43) hooks when the components are added/removed.

If they don't need to handle dynamical add/remove, then just using $children on the glance (like tabbed [used to](https://github.com/rancher/dashboard/commit/e9c2f0e942c35ee2e3e2bd49ee782fa34b901037#diff-5c8b1e033310adfda729578fd1282e1d) do) also works and is simpler.